seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumJSEnv.scala

```diff
@@ -118,26 +118,6 @@ object SeleniumJSEnv {
   }
 
   object Config {
-    private val defaultFactory = {
-      val factory = new DefaultDriverFactory()
-
-      def r(caps: Capabilities, clazz: Class[_ <: WebDriver]) =
-        factory.registerDriverProvider(new DefaultDriverProvider(caps, clazz))
-
-      import org.openqa.{selenium => s}
-
-      r(DesiredCapabilities.firefox(), classOf[s.firefox.FirefoxDriver])
-      r(DesiredCapabilities.chrome(), classOf[s.chrome.ChromeDriver])
-      r(DesiredCapabilities.internetExplorer(), classOf[s.ie.InternetExplorerDriver])
-      r(DesiredCapabilities.edge(), classOf[s.edge.EdgeDriver])
-      r(DesiredCapabilities.operaBlink(), classOf[s.opera.OperaDriver])
-      r(DesiredCapabilities.safari(), classOf[s.safari.SafariDriver])
-      r(DesiredCapabilities.phantomjs(), classOf[s.phantomjs.PhantomJSDriver])
```

Comment:
`phantomjs` has also moved it's location somehow. We are not affected by this, but forcing the update on downstream users might affect them :-/